bitbots_vision_tools/scripts/extract_from_rosbag.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+import rosbag
+import argparse
+import os
+from sensor_msgs.msg import Image
+import cv_bridge
+import cv2
+from cv_bridge import CvBridge
+import numpy as np
+
```

Comment:
Please include the following lines, so this tool also works under py3
```suggestion

try:
    input = raw_input
except NameError:
    pass

```